partner_gender/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Partner Gender
+==============
+
+This module was written to extend the functionality of `base_contact`_ to
```

Comment:
_base_contact_ is intended for another thing: having a notion of contact, that can be attached to multiple companies, but this concept has been misfigured through versions, and with the new v7 partner model, it remains in a grey land which overlaps some expected functionalities from new model (contacts exists already as res.partner with a parent defined).

So all should be _base_partner_*, and better partner_*, because there are plans to dettach partner from base module.